### DISCUSSION TITLE
In this sentence, does 作业 refers to school homework?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1125] [Q&A]
In the Spanish version of the book, "deberes" is shown as the Spanish translation of 作业. In Spanish, "deberes" is a generic word that has a similar meaning to "responsibilities", so "deberes" could include housework,  homework, etc.. In Spanish, the translation of "homework" is "tarea" or "tareas de la escuela", but sometimes Spanish speakers say "deberes de la escuela".

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/b676ff50-0c9b-467b-b34e-5f13cad79296)

In the text, the word 作业 appears and I'm not sure if this refers to homework or housework. I thought that it could mean "housework", because kids can also have other responsibilities in their homes. For example, organizing their room, washing the dishes, cleaning their bathroom, etc.

https://github.com/rdrg109/subtitles-npcr/blob/03e602c945b676d2e060ef67ae958c569756845a/sentences/46-text-1.yaml#L104-L113

For reference, here's the definition of 作业 in  现代汉语规范词典 (p. 1767)

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/2e053c93-1d23-41b8-826b-12e0ff0da862)

For the time being, I'll consider "homework" to be the English translation of "作业" throughout the text.

**Comment by rodrigo-morales-1** [comment]
@fhxshrs friendly ping

**Reply by fhxshrs** [reply]
@rdrg109 在这个语境里，我认为是指学校布置的作业，即"tarea" or "tareas de la escuela"

**Reply by rodrigo-morales-1** [reply]
@fhxshrs Ok! Then, I believe the current translation is suitable.
